spec/support/contexts/response_tree.rb

```diff
@@ -29,4 +29,16 @@ def expect_children(node, types, qing_ids, values = nil)
     child_values = children.map { |child| child.is_a?(Answer) ? child.casted_value : nil }
     expect(child_values).to eq values
   end
+
+  def answer_hash(q_id, value, relevant: true, destroy: nil)
```

Comment:
@scttnlsn I've added this method to `contexts/response_tree.rb` to make setting up the web response hashes easier. I might keep tweaking it, but letting you know in case it, or something like it, might be useful for building expectations.